buildkite/scripts/setup-database-for-archive-node.sh

```diff
@@ -0,0 +1,8 @@
+user=$1
```

Comment:
can you add to the top?

```bash
#!/bin/bash

set -eou pipefail
```

Reply:
Sure.

Reply:
Also can you add the `#!/bin/bash` to the first line?